frontend/src/app/firewood/forests/order-confirmation/order-confirmation.component.ts

```diff
@@ -0,0 +1,248 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
This looks like it was taken from the `buy-firewood-permit.component`/ `tree-application-form.component. Do we need every one of these methods? I know some of them ended up being obsolete on the buy-firewood page.

I was wondering if things like showRulesForm(), updateTotalCost() and checkSeasonStartDate(forest) might not be needed here.

Reply:
Yes, there are a lot of functions here that will potentially need to be removed once we get the pertinent information, such as if all forests are always open, in which case we won't need any of the Season Date related functions.

Once we get that information we can circle back and clean that out. If there are any you see right now that are for sure obsolete, please add a comment to the specific line and we'll get it taken care of.